esp/services/espcontrol/ws_espcontrolservice.cpp

```diff
@@ -223,7 +222,20 @@ bool CWSESPControlEx::onGetLoggingSettings(IEspContext& context, IEspGetLoggingS
             throw MakeStringException(ECLWATCH_INTERNAL_ERROR, "Failed to access container.");
 
         StringBuffer logRequests;
-        resp.setLoggingLevel(m_container->getLogLevel());
+
+        LogLevel level = m_container->getLogLevel();
+
+#ifdef _CONTAINERIZED
+        switch (level)
+        {
+            case ErrMsgThreshold:   level = LogNone;    break;
+            case WarnMsgThreshold:  level = LogMin;     break;
+            case InfoMsgThreshold:  level = LogNormal;  break;
+            case DebugMsgThreshold: level = LogMax;     break;
+        }
```

Comment:
Is this an inclusive mapping, or could something else (like progress) go unmapped?